specs/core/0_fork-choice.md

```diff
@@ -60,8 +66,10 @@ class LatestMessage(object):
 @dataclass
 class Store(object):
     time: uint64
+    genesis_time: uint64
     justified_checkpoint: Checkpoint
     finalized_checkpoint: Checkpoint
+    queued_justified_checkpoints: List[Checkpoint, 2**40] = field(default_factory=list)
```

Comment:
- `2**40` is a magic number
- List with a limit is unnecessary, it's not a SSZ container. `Sequence[Checkpoint]` works. 
- The entry with the highest epoch is only ever used, and then the list is flushed. Why not change it to a single checkpoint and update with `x = max(old.epoch, new.epoch)`

Reply:
Meant to address that before review. whoops

Can't use `Sequence` because it is immutable! 
I can import `typing` and use `typing.List`. It's a bit confusing because we overroad the built in type..

Reply:
good call on just keeping one value. will do